### PR TITLE
Objects Sorting

### PR DESCRIPTION
**Pull request by crimsonred**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## [v4.6.4](https://github.com/pubnub/go/tree/v4.6.5)
```
Comment (davidnub) [inline CHANGELOG.md]: Incorrect copy paste, version should be `v4.6.5`

File: endpoints.go [hunk 2]
```diff
@@ -128,14 +138,18 @@ func buildURL(o endpointOpts) (*url.URL, error) {
...
+				stringifiedQuery += fmt.Sprintf("%s=%s", k, value)
+			} else {
+				stringifiedQuery += fmt.Sprintf("%s=%s&", k, value)
+			}
+			j++
 		}
 
 		i++
 	}
+	//fmt.Println("stringifiedQuery0:", stringifiedQuery)
```
Comment (davidnub) [inline endpoints.go]: Remove?

Comment (crimsonred) [reply endpoints.go]: yep, not ready yet

**Comment by crimsonred** [comment]
All issues (and more) fixed @davidnub

File: examples/cli/cli_demo.go [hunk 3]
```diff
@@ -53,14 +53,18 @@ func connect() {
...
+
 	config.Log = infoLogger
 	config.Log.SetPrefix("PubNub :->  ")
-	config.PublishKey = "demo"
-	config.SubscribeKey = "demo"
-
-	//config.FilterExpression = "name="
+	config.PublishKey = "demo-36"
+	config.SubscribeKey = "demo-36"
+	config.PublishKey = "pub-c-3ed95c83-12e6-4cda-9d69-c47ba2abb57e"
```
Comment (davidnub) [inline examples/cli/cli_demo.go]: Why do we have 2 sets of Pub/Sub keys?

File: examples/cli/cli_demo.go [hunk 4]
```diff
@@ -53,14 +53,18 @@ func connect() {
...
+	config.SubscribeKey = "sub-c-26a73b0a-c3f2-11e9-8b24-569e8a5c3af3"
 
-	config.CipherKey = "enigma"
+	//pamConfig = pubnub.NewConfig()
```
Comment (davidnub) [inline examples/cli/cli_demo.go]: Remove?

File: examples/cli/cli_demo.go (hunk 3 above)
Comment (crimsonred) [reply examples/cli/cli_demo.go]: removed

File: examples/cli/cli_demo.go (hunk 4 above)
Comment (crimsonred) [reply examples/cli/cli_demo.go]: yes done

**Pull request merged**